data/ky/retired/Morgan-McGarvey-9983a801-0a07-4d17-bca0-afb9576ebe69.yml

```diff
@@ -7,7 +7,8 @@ image: https://legislature.ky.gov/Legislators%20Thumbnail%20Images/senate119.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
```

Comment:
I see the PR says it is for MA. Is this update on a retired person for KY meant to be part of the PR?

Reply:
it is not! thank you for catching that

Reply:
LOL ok sorry im so bad at github...im just going to create a new PR